docker-compose.abuse.yml

```
@@ -0,0 +1,34 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  abuse:
+    build:
+      context: ./docker/abuse
+      dockerfile: Dockerfile
+    container_name: abuse
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - ABUSE_LOG_LEVEL=${ABUSE_LOG_LEVEL}
+      - ABUSE_MAILBOX=${ABUSE_MAILBOX}
+      - ABUSE_SPONSOR=${ABUSE_SPONSOR}
+      - BLOCKER_API_URL=${BLOCKER_API_URL}
+      - BLOCKER_AUTH_HEADER=${BLOCKER_AUTH_HEADER}
+      - EMAIL_SERVER=${EMAIL_SERVER}
+      - EMAIL_USERNAME=${EMAIL_USERNAME}
+      - EMAIL_PASSWORD=${EMAIL_PASSWORD}
+      - MONGO_CONNECTIONSTRING=${MONGO_CONNECTIONSTRING}
+    networks:
+      shared:
+        ipv4_address: 10.10.10.110
```

10.10.10.110 is used by `blocker`
```suggestion
        ipv4_address: 10.10.10.120
```

---

updated